### PR TITLE
test

### PR DESCRIPTION
**Pull request by OleksandrFEdev**
- [DEMO LINK](https://OleksandrFEdev.github.io/layout_catalog/)
- [TEST REPORT LINK](https://OleksandrFEdev.github.io/layout_catalog/report/html_report/)

The last 2 tests failed, but visually there are no problems.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,8 +5,316 @@
...
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Smartphones</a>
+          </li>
```
Comment (IvanFesenko) [inline src/index.html:36]: don`t forget to add empty line between sibling tags

File: src/index.html [hunk 2]
```diff
@@ -5,8 +5,316 @@
...
+          <li class="nav__item">
+            <a class="nav__link"
+            href="#"
+            data-qa="nav-hover"
+            >
```
Comment (IvanFesenko) [inline src/index.html:41]: Pay attention on indents. Fix all similar cases
```suggestion
            <a class="nav__link"
              href="#"
              data-qa="nav-hover"
            >
```

File: src/index.html [hunk 3]
```diff
@@ -5,8 +5,316 @@
...
+    <main class="catalog">
+      <div class="card" data-qa="card">
+        <img class="card__img"
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        >
+        <h2 class="card__text">
+          <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        </h2>
+        <div class="card__product"><p>Product code: 195434</p></div>
```
Comment (IvanFesenko) [inline src/index.html:69]: Do not use div for text content - use semantic tags instead,  p for example

File: src/styles/block/card.scss [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
+  background: url(/images/star-active.svg) no-repeat center;
+}
+
```
Comment (IvanFesenko) [inline src/styles/block/card.scss:61]: Stars should work whit modifiers (1-5)

File: src/styles/block/card.scss [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+.card__price-label {
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__money {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: bold;
```
Comment (IvanFesenko) [inline src/styles/block/card.scss:77]:
```suggestion
  font-weight: 700;
```

File: src/styles/block/card.scss [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+  display: flex;
+}
+
+.stars__star {
+  background: url(/images/star.svg) no-repeat center;
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+}
+
```
Comment (IvanFesenko) [inline src/styles/block/card.scss:56]: Move stars block to separate file

File: src/styles/block/catalog.scss [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+  margin-top: 50px;
+}
+
+.card:not(:nth-child(4n)) {
+  margin-right: $card-side-margin;
+}
+
+.card:nth-last-child(-n + 4) {
+  margin-top: 46px;
+}
```
Comment (IvanFesenko) [inline src/styles/block/catalog.scss:18]: If its catalof block file - why .card are here?

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Check my comments

File: src/index.html (hunk 3 above)
Comment (OleksandrFEdev) [reply src/index.html:76]: if I use a semantic tag here, the positioning gets messed up

File: src/index.html [hunk 8]
```diff
@@ -5,8 +5,323 @@
...
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
```
Comment (volodymyr-soltys97) [inline src/index.html:45]: If we have 3 or more attributes, then all of them must be on a new line
```suggestion
            <a
              class="nav__link"
              href="#"
              data-qa="nav-hover"
            >
```

File: src/index.html [hunk 9]
```diff
@@ -5,8 +5,323 @@
...
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-label">Price:</p>
+          <p class="card__money">$2,199</p>
+        </div>
+        <a
+        class="card__button"
+        href="#"
+        data-qa="card-hover"
```
Comment (volodymyr-soltys97) [inline src/index.html:96]:
```suggestion
          class="card__button"
          href="#"
          data-qa="card-hover"
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
But let's fix your code with a code style guide https://codeguide.academy/html-css.html#css-order

File: src/styles/block/card.scss [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+.card__product {
+  margin-bottom: 16px;
+}
+
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
```
Comment (mykhalenych) [inline src/styles/block/card.scss:38]:
```suggestion
  align-items: center;
```

File: src/index.html [hunk 11]
```diff
@@ -5,8 +5,324 @@
...
+
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__stars-text"><p>Reviews: 5</p></div>
```
Comment (mykhalenych) [inline src/index.html:87]: change to p and delete child p

File: src/styles/block/card.scss [hunk 12]
```diff
@@ -0,0 +1,85 @@
+html {
```
Comment (mykhalenych) [inline src/styles/block/card.scss:1]: move html to main.scss file

**Review by mykhalenych — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 13]
```diff
@@ -5,8 +5,324 @@
...
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
```
Comment (Viktor-Kost) [inline src/index.html:50]:
```suggestion
          </li>
          
          <li class="nav__item">
```

File: src/styles/block/stars.scss [hunk 14]
```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url(/images/star.svg) no-repeat center;
+  margin-right: 4px;
```
Comment (Viktor-Kost) [inline src/styles/block/stars.scss:7]: last star shouldn't have margin. Use `:not(:last-child)`

File: src/index.html [hunk 15]
```diff
@@ -5,8 +5,324 @@
...
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div class="card" data-qa="card">
+        <img class="card__img"
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        >
```
Comment (Viktor-Kost) [inline src/index.html:73]:
```suggestion
        <img 
          class="card__img"
          src="images/imac.jpeg"
          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
        >
```

File: src/styles/block/catalog.scss [hunk 16]
```diff
@@ -0,0 +1,18 @@
+.catalog {
```
Comment (Viktor-Kost) [inline src/styles/block/catalog.scss:1]: top line of cards shouldn't have margin-top

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

**Review by slavanosatskyi — APPROVED** [review]

**Pull request closed without merging**